src/coreclr/vm/syncblk.cpp

```diff
@@ -2549,14 +2554,9 @@ BOOL AwareLock::EnterEpilogHelper(Thread* pCurThread, INT32 timeOut)
         QueryPerformanceCounter(&startTicks);
 
         // Fire a contention start event for a managed contention
-        FireEtwContentionStart_V1(ETW::ContentionLog::ContentionStructs::ManagedContention, GetClrInstanceId());
+        FireEtwContentionStart_V2(ETW::ContentionLog::ContentionStructs::ManagedContention, GetClrInstanceId(), (ObjectID)OBJECTREFToObject(obj), (ThreadID)m_HoldingThread);
```

Comment:
`ObjectID` and `ThreadID` are ICorProfiler concepts, we shouldn't use them here. A cast shouldn't be necessary at all, if the compiler complains then the cast should be using the data types defined in clretwall.man, i.e. `(UINT64)`.

Reply:
done https://github.com/dotnet/runtime/pull/72627/commits/5fb79fdff0d498b6e456d5a85bc9c9d97d5fb821